README.md

```diff
@@ -1,101 +1,140 @@
 <!--lint disable awesome-heading-->
+
 # Awesome Stylelint [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
 
 [<img src="https://raw.githubusercontent.com/stylelint/stylelint/master/identity/stylelint-icon-white-512.png" width="160" align="right" alt="stylelint">](https://stylelint.io/)
 
-> A list of awesome Stylelint configs, plugins, etc.
+> A list of awesome Stylelint configs, plugins, integrations etc.
 
 ## Contents
 
+- [Analysis platforms](#analysis-platforms)
 - [Configs](#configs)
+- [Custom syntaxes](#custom-syntaxes)
+- [Editor integrations](#editor-integrations)
 - [Formatters](#formatters)
 - [Plugins](#plugins)
+- [Task runners](#task-runners)
+- [Testing](#testing)
 - [Tools](#tools)
 - [Tutorials](#tutorials)
 
+## Analysis platforms
+
+- [codacy-stylelint](https://github.com/codacy/codacy-stylelint) - Codacy integration.
+- [codeclimate-stylelint](https://github.com/gilbarbara/codeclimate-stylelint) - Code Climate integration.
+- [mega-linter](https://nvuillam.github.io/mega-linter) - Integrated into Mega-Linter.
+- [putout](https://github.com/coderaiser/putout) - Integrated into Putout.
+- [reviewdog/action-stylelint](https://github.com/reviewdog/action-stylelint) - Reviewdog GitHub Action.
+
 ## Configs
 
-- [stylelint-recommended](https://github.com/stylelint/stylelint-config-recommended) - The recommended shareable config for Stylelint.
-- [stylelint-recommended-scss](https://github.com/stylelint-scss/stylelint-config-recommended-scss) - Same as above, when using SCSS.
```

Comment:
[ask] Are the recommended configs missing?

Reply:
Yes, as they can be reached via the READMEs of the standard configs. Ironically, we don't recommend people use the recommended config. The standard configs are a better fit for the majority of users.